src/components/Home/Home.js

```diff
@@ -1,22 +1,23 @@
-import React, { useEffect } from "react";
-import MovieListing from "../MovieListing/MovieListing";
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import MovieListing from '../MovieListing/MovieListing';
 
-import { useDispatch } from "react-redux";
 import {
   fetchAsyncMovies,
   fetchAsyncShows,
-} from "../../features/movies/movieSlice";
+} from '../../redux/movies/movieSlice';
+
 const Home = () => {
```

Comment:
- [ ] Based on the requirements of this project, we are supposed to follow the design guidelines including Typography(font face, size, and weight) and layout(composition and space between elements). Let's kindly check the difference in the compared screenshot below and make the necessary changes.

| Yours | Design guidelines |
| --- | --- |
| ![Screenshot from 2023-01-13 05-17-44](https://user-images.githubusercontent.com/63087491/212236324-0c9ecd24-f7c2-408e-bb0e-a28ade7b0408.png) | ![Screenshot from 2022-12-02 19-27-09](https://user-images.githubusercontent.com/63087491/210885093-fa407447-4b3f-4e0e-96a3-a113e52746a0.png) |

NB. Having a nice design increases the chances of a job as recruiters will be checking projects on GitHub.